server/services/voip/connector/asterisk/ami/PJSIPEndpoint.ts

```diff
@@ -28,6 +28,8 @@
  * 7. Important to note that the intermediate events containing a result part for an execution of a particular command
  *    have same actionid, which is received by this class as a successful execution of a command in actionResultCallback.
  */
+import { _ } from 'meteor/underscore';
```

Comment:
In here, you should be able to use `underscore` instead of the meteor package.

Reply:
Fixed.